README.md

```diff
@@ -138,6 +120,38 @@ Notable features:
 [^5b]: https://www.ubicloud.com/blog/ubicloud-hosted-arm-runners-100x-better-price-performance
 [^5c]: https://github.com/ubicloud/ubicloud/blob/main/routes/web/webhook/github.rb
 
+### RunsOn
+
+[RunsOn](https://runs-on.com/) is a self-hosted GitHub Actions runner tool that runs on your own AWS account. It's **free for non-commercial usage**, and comes at a €300 flat yearly fee for commercial projects.
+
+Pricing [here](https://runs-on.com/).
+
+Notable features:
```

Comment:
I would also add the "1-line change" sentence as for the others, and mention the runtime flexibility in the choice of instance types, disk size, ssh access, and most importantly custom base images

Reply:
Good stuff, I'll make some modifications.